components/automate-chef-io/data/docs/api_chef_automate/iam/v2beta/tokens.swagger.json

```diff
@@ -188,6 +188,9 @@
         }
       }
     },
+    "chef.automate.api.iam.v2beta.ResetAllTokenProjectsResp": {
+      "type": "object"
+    },
     "chef.automate.api.iam.v2beta.Token": {
       "type": "object",
       "properties": {
```

Comment:
Some of these changes shouldn't be in this PR. I just merged #2004 which should clean this up if you rebase.